source/adapters/level_zero/v2/event_provider_normal.cpp

```diff
@@ -37,7 +42,7 @@ provider_pool::provider_pool(ur_context_handle_t context,
   }
 
   ZE2UR_CALL_THROWS(zeEventPoolCreate,
-                    (context->ZeContext, &desc, 1,
+                    (context->hContext, &desc, 1,
```

Comment:
I think we should have getters for ze handles. But we can do that later.

Reply:
Yes, agree. I actually have the getters implemented already on a branch with enqueue implementation because we need them to be able to reuse some code from legacy adapter (like program.cpp)